include/mxnet/ndarray.h

```diff
@@ -121,7 +121,7 @@ class NDArray {
     raw_shape[0] = length;
     MSHADOW_TYPE_SWITCH(dtype_, DType, {
       res = TBlob(static_cast<DType*>(ptr_->shandle.dptr)
-        + offset_ + offset, raw_shape, ptr_->shandle.ctx.dev_mask());
+        + offset_ + offset, raw_shape, ptr_->shandle.ctx);
```

Comment:
the original tblob::chunk has one offset, also raw_data can accept an offset as an argument